league/manager/models.py

```diff
@@ -173,6 +177,9 @@ class EventType(models.TextChoices):
     MATCH_LOST = "Match has been lost"
     MATCH_DRAW = "Match has been drawn"
     MATCH_GOAL = "Goal has been scored"
+    MATCH_FIRST_YELLOW = "First yellow card"
+    MATCH_SECOND_YELLOW = "Second yellow card"
+    MATCH_RED = "Red card"
```

Comment:
unnecessary change, we won't be using cards since it's not a football league, but a table football one